packages/apps/src/microsoft/teams/apps/app_process.py

```diff
@@ -42,6 +42,7 @@ def __init__(
         default_connection_name: str,
         http_client: Client,
         token: AppTokens,
+        graph_token_refresher: Optional[Callable[[Optional[str]], Awaitable[Optional[str]]]] = None,
```

Comment:
This is making me think we need another simple clsss that can manage tokens for us. Wdyt?